app/controllers/webhooks/instagram_controller.rb

```diff
@@ -25,6 +25,6 @@ def events
   private
 
   def valid_instagram_token?(token)
-    token == ENV['IG_VERIFY_TOKEN']
+    token == GlobalConfigService.load('FB_VERIFY_TOKEN', '')
```

Comment:
No No it should be `GlobalConfigService.load('IG_VERIFY_TOKEN', '')`

Reply:
My bad. Fixed.